ui/components/multichain/pages/page/components/header/index.scss

```diff
@@ -0,0 +1,33 @@
+.multichain-page-header {
+  $height-screen-sm-max: 100%;
+  $width-screen-sm-min: 85vw;
+  $width-screen-md-min: 80vw;
+  $width-screen-lg-min: 62vw;
+
+  flex-flow: column nowrap;
+  z-index: 55;
```

Comment:
Can we keep all z-index values in `ui/css/design-system/z-index.scss` and use a sass variable here so we can keep track of all z-indexes?